patches/tModLoader/Terraria/Recipe.cs.patch

```diff
@@ -150,14 +156,23 @@
  	{
  		if (customShimmerResults == null)
  			customShimmerResults = new List<Item>();
-@@ -120,9 +_,10 @@
+@@ -120,9 +_,19 @@
  		item.SetDefaults(itemType);
  		item.stack = itemStack;
  		customShimmerResults.Add(item);
 -		return item;
 +		return this;
  	}
  
++	/// <summary>
++	/// Sets the priority of this recipe over other recipes for the same item, highest first, defaults to 0
++	/// </summary>
++	public Recipe SetShimmerPriority(int priority)
```

Comment:
TML does not accept numerical priorities, because we want to avoid priority fighting. I suggest we run this PR without the priority system, and just use recipe ordering to start with.

If specific decrafting priorities are required, they need to be specified in a relational manner with before/after constraints, similar to all other sorting in tML (including recipe sorting)

Reply:
👍